widget/tabcontainer.go

```diff
@@ -163,10 +144,42 @@ func (t *TabContainer) buildTabBar(buttons []fyne.CanvasObject) *fyne.Container
 	return tabBar
 }
 
+// Append adds a new CanvasObject to the end of the group
+func (t *TabContainer) Append(item *TabItem) {
+	r := cache.Renderer(t).(*tabContainerRenderer)
+	t.Items = append(t.Items, item)
+	r.tabBar.Objects = append(r.tabBar.Objects, t.makeButton(item))
+
+	t.Refresh()
+}
+
+// Remove tab by value
+func (t *TabContainer) Remove(item *TabItem) {
+	r := cache.Renderer(t).(*tabContainerRenderer)
+
+	for index, existingItem := range t.Items {
+		if existingItem == item {
+			t.Items = append(t.Items[:index], t.Items[index+1:]...)
```

Comment:
Would it not be a little neater to call `RemoveIndex(index)` here to remove the duplication?